packages/next/src/server/image-optimizer.ts

```diff
@@ -534,7 +534,11 @@ export async function imageOptimizer(
   const { isAbsolute, href, width, mimeType, quality } = paramsResult
 
   if (isAbsolute) {
-    const upstreamRes = await fetch(href)
+    const upstreamRes = await fetch(href, {
+      headers: {
+        "Accept": "image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8"
```

Comment:
```suggestion
        Accept: 'image/avif,image/webp,*/*'
```

I think "Firefox 92 and later" got it right so lets use that

Reply:
```suggestion
        Accept: 'image/avif,image/webp,image/*,*/*'
```

Actually, the `image/*` is probably good to have in there too.